static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js

```diff
@@ -300,13 +300,18 @@ const buildPageTargetting = (
     // This can be removed once we get sign-off from third parties who prefer to use appNexusPageTargeting.
     page.pageAdTargeting = pageTargeting;
 
-    return pageTargeting;
+    return Promise.resolve(pageTargeting);
 };
 
 const getPageTargeting = (): { [key: string]: mixed } => {
     if (Object.keys(myPageTargetting).length !== 0) return myPageTargetting;
```

Comment:
this is wrong now, I think it should be:

```suggestion
const getPageTargeting = (): Promise<{ [key: string]: mixed }> => {
    if (Object.keys(myPageTargetting).length !== 0) return Promise.resolve(myPageTargetting);
```

and then because getPageTargeting is exported, everything that uses it will need to be updated?